src/main/js/LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018 The Finnish National Board of Education - Opetushallitus
+Copyright (c) 2019 The Finnish National Board of Education - Opetushallitus
```

Comment:
I remember that english version has changed to Finnish National Agency for Education